Documentation Improvements

zeelot opened the pull request:
Making a pull request for the sake of tracking these changes. I'll be merging it in after fixing a group of documentation pages (accuracy and clarity).

Revamp Ticket: http://dev.kohanaframework.org/issues/4723
Cookie Salt Ticket: http://dev.kohanaframework.org/issues/4644

zeelot committed:
reword the install.md documentation page (refs #4723 refs #4644)

rjd22 commented on guide/kohana/install.md:
```diff
@@ -1,45 +1,64 @@
+
+ - Define a salt for the `Config` class.
+~~~
+Cookie::$salt = 'd131dd02c5e6eec4693d9a0698aff95c2fcab58712467eab4004583eb8fb7f89'
```
I would change the string to [really-long-cookie-salt-here] and link then to http://www.random.org/strings/ to generate their own random string.

zeelot committed:
Better Cookie::$salt docs (refs #4723 refs #4644)

zeelot commented:
Made that change, thanks.

zeelot committed:
Fix documentation typo for Cookie::$salt docs (refs #4723 refs #4644)

rjd22 commented:
I think it's good enough to merged in. Don't you agree?

zeelot commented:
I want to improve many other pages as part of this, including removing the silly "Getting Started" submenu as that is just hiding pages under an arbitrary category.

maddes committed:
refs #4723 - Chopping off bad practice example.

Zeelot: "The recommended way is to just build the full url from a route without recycling the one in Request.
you don't know which route the user got to your page with, trying to recycle parameters from it is asking for trouble"

So, removing the example...

zeelot committed:
Merge pull request #372 from maddes/patch-2

refs #4723 - Chopping off bad practice example.
